transfer/args.go

```diff
@@ -19,12 +19,12 @@ const (
 )
 
 // ParseArgs parses the given args.
-func ParseArgs(lines []string) (map[string]string, error) {
-	args := make(map[string]string, 0)
-	for _, line := range lines {
+func ParseArgs(parts []string) (Args, error) {
+	args := make(Args, 0)
+	for _, line := range parts {
 		parts := strings.SplitN(line, "=", 2)
 		if len(parts) != 2 {
-			return nil, fmt.Errorf("invalid argument: %q", line)
+			continue
```

Comment:
Why are you now ignoring parts that are not key-value-pairs? Is there a specific use case for this?

Reply:
I believe "arguments" was referring to the `*argument` production before the `*oid-line` production, not non-key-value-pairs within the `oid-line` production. IMO, non-key-value pairs within an `oid-line` production (after the `oid` and `size` arguments) should still trigger an error.

Reply:
I believe `*argument` and `oid-line` arguments use the same key-value pair format. Unknown arguments depend on the backend implementation. Perhaps you're right though, according to the specs `key=[value]`, an argument _must_ have a key followed by an `=` sign and an optional value. Thus, we should return an error here